src/panels/AssetsFolderPanel/util.ts

```diff
@@ -0,0 +1,24 @@
+import { Asset, Folder } from './types';
+
+export function getCurrentFolder(root: Folder, path: string[]): Folder {
```

Comment:
```suggestion
export function currentFolder(root: Folder, path: string[]): Folder {
```

For naming consistency with the main repository